src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tr');
```

Comment:
The `querySelector` method will only select the first 'tr' element in the table. If you intend to select all rows, you should use `querySelectorAll` instead.